Documentation/guides/AndroidMavenLibrary.md

```diff
@@ -0,0 +1,59 @@
+# AndroidMavenLibrary
+
+Note: This feature is only available in .NET 9+.
+
+## Description
+
+`<AndroidMavenLibrary>` allows a Maven artifact to be specified which will automatically be downloaded and added to a .NET Android binding project. This can be useful to simplify maintenance of .NET Android bindings for artifacts hosted in Maven.
+
+## Specification
+
+ A basic use of `<AndroidMavenLibrary>` looks like:
+
+```xml
+<!-- Include format is {GroupId}:{ArtifactId} -->
+<ItemGroup>
+  <AndroidMavenLibrary Include="com.squareup.okhttp3:okhttp" Version="4.9.3" />
+</ItemGroup>
```

Comment:
Can we show an example of how you'd copy paste from here:

https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp/4.9.3

And then turn it into our XML/MSBuild version?

Reply:
> https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp/4.9.3

mvnrepository is agregate repo and most packages point to MavenCentral repo.

Pobst used proper maven name for `artifactId` which is `groupId:artifactId` 

https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#minimal-pom

It would be nice to have FQ Maven Id with version too: `groupId:artifactId:version` then MsBuild `Version` attribute is not needed.

Reply:
The rationale for a separate `Version` attribute is to mirror `<PackageReference>` and to allow MSBuild central versioning strategies to work, like:

```xml
<AndroidMavenLibrary Include="com.squareup.okhttp3:okhttp" Version="$(OKHTTP3_VERSION)" />
```

or

```xml
<AndroidMavenLibrary Update="com.squareup.okhttp3:okhttp" Version="5.0.0" />
```

I suppose we could allow either format, but let's start with this one and see if there is any demand for alternative formats.